docs/mysql.md

```diff
@@ -9,6 +9,15 @@ First, install the `symfony/orm-pack` package as described: `docker compose exec
 Change the database image to use MySQL instead of PostgreSQL in `compose.yaml`:
 
 ```diff
+services:
+    php:
+    environment:
+...
+-     DATABASE_URL: postgresql://${POSTGRES_USER:-app}:${POSTGRES_PASSWORD:-!ChangeMe!}@database:5432/${POSTGRES_DB:-app}?serverVersion=${POSTGRES_VERSION:-15}&charset=${POSTGRES_CHARSET:-utf8}
++     DATABASE_URL=mysql://${MYSQL_USER:-app}:${MYSQL_PASSWORD:-!ChangeMe!}@database:3306/${MYSQL_DATABASE:-app}?serverVersion=${MYSQL_VERSION:-8}&charset=${MYSQL_CHARSET:-utf8mb4}
+...
+
+...
```

Comment:
Placeholders in comments with indentation
```suggestion
      # ...
-     DATABASE_URL: postgresql://${POSTGRES_USER:-app}:${POSTGRES_PASSWORD:-!ChangeMe!}@database:5432/${POSTGRES_DB:-app}?serverVersion=${POSTGRES_VERSION:-15}&charset=${POSTGRES_CHARSET:-utf8}
+     DATABASE_URL=mysql://${MYSQL_USER:-app}:${MYSQL_PASSWORD:-!ChangeMe!}@database:3306/${MYSQL_DATABASE:-app}?serverVersion=${MYSQL_VERSION:-8}&charset=${MYSQL_CHARSET:-utf8mb4}
      # ...

  # ...
```

see
- https://github.com/dunglas/symfony-docker/blob/d572404fa3b6b73abc729897be4fbcafd0725780/docs/production.md?plain=1#L105-L109
- https://github.com/dunglas/symfony-docker/blob/d572404fa3b6b73abc729897be4fbcafd0725780/docs/tls.md?plain=1#L31-L37